vllm/worker/hpu_model_runner.py

```diff
@@ -937,8 +1011,7 @@ def _prepare_prompt(
         assert max_query_len > 0
 
         max_prompt_len = max(
-            find_bucket(max_query_len,
-                        self.bucketing_global_state.prompt_seq_bucket_cfg),
+            self.bucketing_ctx.get_padded_prompt_seq_len(max(seq_lens)),
```

Comment:
Hi @kdamaszk, should it be `self.bucketing_ctx.get_padded_prompt_seq_len(max_query_len)` ? Will it affect the `prefix_caching` feature?